### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by deepankur797** [dhruvmewada15/terraform-assignment3#1]
1. You can get the ami details using the ```data``` resource. 
2. You are exposing the password in your database module .

**Comment by anjuls** [comment]
Modules should be an independent entity with no hard coded values. It is assumed that I should be able to reuse the module and should be documented, what is the input and what is the output expected out of it.

**Comment by dhruvmewada15** [comment]
Thank you @deepankur797  @anjuls for the review. 
@deepankur797 I will find other ways for passing the password. One solution I found is passing the password through env variables. Is that fine?
@anjuls I will fix the documentation part but can you give me more information on hard coded values I have used in the modules where I shoud not have used?

**Comment by deepankur797** [comment]
Yes you can use environment variables. You can also use a lot of other stuff as well like terraform cloud. we discussed this on the following thread few weeks back 

https://infracloud.slack.com/archives/CGEUEU2RE/p1613976841009500

**Comment by anjuls** [comment]
@dhruvmewada15 I am talking about `region` which is hardcoded in modules. This is an example.
